flowzone.yml

```diff
@@ -1567,6 +1589,7 @@ jobs:
             echo "enabled=true" >> "${GITHUB_OUTPUT}"
             echo "private=$(jq -r '.private' package.json)" >> "${GITHUB_OUTPUT}"
             echo "docs=$(jq -r '.scripts | has("doc")' package.json)" >> "${GITHUB_OUTPUT}"
+            echo "sbom=true" >> "${GITHUB_OUTPUT}" # TODO: evaluate the need to offer an option to disable SBOM generation
```

Comment:
What are the chances of this process breaking on untested repos? Do we need the option to turn it off, or should it be opt-in at first.

On the other hand, we have very few python and cargo projects so this seems low risk.

Reply:
I'd say low but non-null

I tested python on a copy of balena-cli (works fine), but didn't test the cargo one outside this repo.

Reply:
Here's the result for the latest test atm: https://github.com/product-os/flowzone/actions/runs/9619401059